contrib/completion/bash/docker

```diff
@@ -270,6 +270,7 @@ __docker_log_drivers() {
 		json-file
 		none
 		syslog
+		splunk
```

Comment:
Please insert this in alphabetical order. This also applies to the other additions.